db/seeds.rb

```diff
@@ -82,24 +82,30 @@ def log(msg)
 }')
 
 log "7. Seeding Users"
+# Receivers is a temporary table.  When we get the data from Highlands SSO we'll remove the receivers table.
```

Comment:
I dont know if we are going to `Highlands SSO we'll remove the receivers table.`. Thoughts @knewter ?